Resources/Prototypes/SimpleStation14/Catalog/Fills/Items/briefcases.yml

```diff
@@ -0,0 +1,11 @@
+# let me know when money actually becomes useful to normal people
+# - type: entity
+#   id: BriefcaseBrownCash
+#   name: brown briefcase
+#   parent: BriefcaseBrown
+#   suffix: Cash
+#   components:
+#   - type: StorageFill
+#     contents:
+#       - id: SpaceCash10000
+#         amount: 2
```

Comment:
It being filled with two things is lame >:P
Needs to be like 50 500s. But also fills need a way to do more than one of an entity without doing it by fuckin adding it 50 times

Reply:
yeah I suppose I could change it